src/Assets/Asset.php

```diff
@@ -914,7 +914,7 @@ public function maybe_get_min_file( $url ) {
 		// Strip the plugin URL and make this relative.
 		$relative_location = str_replace( $base_url . '/', '', $url );
 
-		if ( $script_debug ) {
+		if ( Config::should_ignore_script_debug() || $script_debug ) {
```

Comment:
```suggestion
		if ( ! Config::should_ignore_script_debug() && $script_debug ) {
```

Shouldn't this be like this ?

Reply:
No, because we are ignoring the script debug I dont care about the second part of the logic right?